src/app/containers/Headings/index.jsx

```diff
@@ -25,7 +22,7 @@ const Headings = {
 };
 
 const GridConstraints = {
-  headline: GridItemConstrainedLarge,
+  headline: GridItemLarge,
   subheadline: GridItemConstrainedMedium,
 };
```

Comment:
Please can you help me understand why we have changed `GridItemConstrainedLarge` to `GridItemLarge`, but not `GridItemConstrainedMedium` to `GridItemMedium`?

Reply:
I was trying to minimize the changes needed for the `Grid` updates to work. The next steps would be to gradually replace the old implementations in small PR's to aid faster review.